src/documentation/sass/OfficeDev/_Icons.scss

```diff
@@ -14,7 +14,7 @@
   font-style: normal;
   font-weight: normal;
   line-height: 1;
-  speak: none;
+//   speak: none;
```

Comment:
What's the purpose of this? Wondering if the change should just be reverted.

Reply:
It was throwing some errors in the build while I was still in the process of converting Gulp but I'll uncomment it, thanks!